crates/ika-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -364,9 +365,14 @@ pub struct AuthorityEpochTables {
     /// The key is the consensus round number,
     /// the value is the dWallet-mpc messages that have been received in that
     /// round.
+    #[default_options_override_fn = "dwallet_mpc_messages_table_default_config"]
     dwallet_mpc_messages: DBMap<Round, Vec<DWalletMPCMessage>>,
-    dwallet_mpc_outputs: DBMap<Round, Vec<DWalletMPCOutputMessage>>,
-    dwallet_mpc_completed_sessions: DBMap<Round, Vec<SessionIdentifier>>,
+
+    #[default_options_override_fn = "dwallet_mpc_outputs_table_default_config"]
+    dwallet_mpc_outputs: DBMap<Round, Vec<DWalletMPCOutput>>,
+
+    #[default_options_override_fn = "dwallet_mpc_computation_completed_sessions_table_default_config"]
+    dwallet_mpc_computation_completed_sessions: DBMap<Round, Vec<SessionIdentifier>>,
```

Comment:
You are saving duplicate information here, there is already the session ID in side the output